lib/brains/webhooks.py

```diff
@@ -30,6 +35,29 @@ def process(self):
 
         log.info('Processing event: {0}'.format(self.webhook.kind))
         method()
+        self.processed = True
+
+    @property
+    def data(self):
+        if not self.processed:
+            return
+
+        subscription = self.get_subscription()
+        try:
+            transaction = self.transactions[0]
```

Comment:
This seems a little fragile. At the very least maybe raise an error if there is more than one transaction?

I think I missed something with transactions. I tried to look this up in Braintree's guide to webhooks but couldn't find it. Maybe ping me on this when you are free? I probably need to understand more about how it works.